kahuna/public/js/upload/prompt/prompt.css

```diff
@@ -5,8 +5,12 @@ file-prompt {
 }
 
 file-prompt file-uploader {
-    display: flex;
-    justify-content: center;
+    display: inline-block;
+    padding: 0 5px;
+}
+
+file-prompt file-uploader .file-uploader__select-files {
+    margin-top: 0;
```

Comment:
Bit annoying that we have to do this. It looks like `margin-top` is baked into the component for positioning in `gr-top-bar` of the search page, which isn't ideal as its coupling page specific styles into components. I tried (quickly) decoupling, but it got messy and made https://github.com/guardian/grid/issues/1172 worse.